helper/helper.go

```diff
@@ -205,29 +202,22 @@ func preloadCreatedPipesForBackup(oidList []int, queuedPipeCount int) {
 	}
 }
 
-func preloadCreatedPipesForRestore(oidWithBatchList []oidWithBatch, queuedPipeCount int) {
+func (h *Helper) preloadCreatedPipesForRestore(oidWithBatchList []oidWithBatch, queuedPipeCount int) {
 	for i := 0; i < queuedPipeCount; i++ {
 		pipeName := fmt.Sprintf("%s_%d_%d", *pipeFile, oidWithBatchList[i].oid, oidWithBatchList[i].batch)
 		pipesMap[pipeName] = true
 	}
 }
 
-func getOidWithBatchListFromFile(oidFileName string) ([]oidWithBatch, error) {
-	oidStr, err := operating.System.ReadFile(oidFileName)
-	if err != nil {
-		logError(fmt.Sprintf("Error encountered reading oid batch list from file: %v", err))
-		return nil, err
-	}
-	oidStrList := strings.Split(strings.TrimSpace(fmt.Sprintf("%s", oidStr)), "\n")
-	oidList := make([]oidWithBatch, len(oidStrList))
-	for i, entry := range oidStrList {
-		oidWithBatchEntry := strings.Split(entry, ",")
-		oidNum, _ := strconv.Atoi(oidWithBatchEntry[0])
-		batchNum, _ := strconv.Atoi(oidWithBatchEntry[1])
+type IHelper interface {
+	createPipe(pipe string) error
```

Comment:
Function to delete a pipe is in the `IRestoreHelper`, while IHelper has the function to create a pipe... Sounds a bit strange. Can't all pipe related functions be under the same interface?